packages/pluggableWidgets/slider-web/src/ui/Slider.scss

```diff
@@ -0,0 +1,14 @@
+.widget-slider {
+    padding: 4px 8px;
+    margin-bottom: 16px;
+
+    &.widget-slider-vertical {
+        height: 100%;
+    }
+}
+
+.widget-slider-no-value {
+    .rc-slider-track {
+        display: none;
+    }
+}
```

Comment:
I don't think this is used, or am I missing something?

Reply:
Backwards compatibility because of https://github.com/mendix/widgets-resources/blob/7a358d813c5289770e892fd260304bb9665faf33/packages/customWidgets/slider-web/src/components/Slider.ts#L60

Reply:
The type could not be restricted before apparently

Reply:
But in new version of widget there no `value` prop

Reply:
@diego-antonelli can you confirm/deny whether this is necessary anymore or not?

Reply:
Not needed anymore. If the widget is not using, lets removed it.